infra/_modules/ai_search/variables.tf

```diff
@@ -105,4 +105,15 @@ variable "indexers_scheduling_interval" {
     services_lifecycle   = "PT1H"
     services_publication = "PT1H"
   }
+}
+
+variable "sku" {
+  type        = string
+  description = "The SKU of the Search Service"
+  default     = "standard"
```

Comment:
May it be "StandardS1" the sku name?

Reply:
Initially I thought the same. However, from the [resource documentation](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/search_service#arguments-reference),  it looks like "standard" is the right one